src/assets/styles/layout.scss

```diff
@@ -75,7 +75,7 @@
 
 // Scrollable Container
 .scrollable-container {
-  overflow-y:scroll;
+  overflow: auto;
```

Comment:
I couldn't stand those scroll bars visible on elements that didn't require scrolling
This will display them if they are required, hide them if not.

Reply:
What happens to horizontal scrolling on narrow screens?

Reply:
Good call, will check. 

If you go down to a mobile size, a horizontal bar will display. 
I could `overflow-y: auto` & `overflow-x: hidden` if you think this is something we should support.

Reply:
Weird, I don't get horiz scrollbars when I make Chrome super narrow.

I'm thinking what you have now is OK.

Actually, there might be a ticket for hiding the scrollbars. Check Zenhub.

Reply:
Cool, 2 birds with 1 stone!

Reply:
Updated 👍  Moved #9282